tests/sfbase.py

```diff
@@ -14,7 +14,14 @@
 
 class SFBaseTest(BaseCase):
 
+    """
+    Set these variables for the properties to use in the other tests
+    """
+
     salesforce_api = "BULK"
+    total_quota = '80'
```

Comment:
both base and sfbase are moving to 95% total_quota

Reply:
changed it